utils/src/main/kotlin/DeclaredLicenseProcessor.kt

```diff
@@ -43,22 +43,25 @@ object DeclaredLicenseProcessor {
         ".dbk", ".html", ".md", ".odt", ".php", ".rtf", ".tex", ".txt"
     )
 
-    fun process(declaredLicense: String): SpdxExpression? {
+    internal fun preprocess(declaredLicense: String): String {
```

Comment:
How about calling this function `trim`?

Reply:
Hmm, I liked preprocess vs. process, and I didn't want to imply that this will never do more than trimming prefixes / suffixes in the future.